internal/ingress/annotations/proxyssl/main.go

```diff
@@ -39,9 +41,73 @@ const (
 
 var (
 	proxySSLOnOffRegex    = regexp.MustCompile(`^(on|off)$`)
-	proxySSLProtocolRegex = regexp.MustCompile(`^(SSLv2|SSLv3|TLSv1|TLSv1\.1|TLSv1\.2|TLSv1\.3)$`)
+	proxySSLProtocolRegex = regexp.MustCompile(`^(SSLv2|SSLv3|TLSv1|TLSv1\.1|TLSv1\.2|TLSv1\.3| )*$`)
```

Comment:
Was this rule intentionally modified?

Reply:
yes, because you can separate protocols by spaces on the configuration. The place where this REGEX is used on the code also changed and not having spaces was a problem